app/invites/server/functions/validateInviteToken.js

```diff
@@ -0,0 +1,34 @@
+import { Meteor } from 'meteor/meteor';
+
+import { Invites, Rooms } from '../../../models';
+
+export const validateInvite = (inviteData, room) => {
+	if (!inviteData) {
+		return false;
+	}
+
+	if (!room) {
+		return false;
+	}
+
+	if (inviteData.expires && inviteData.expires <= Date.now()) {
+		return false;
+	}
+
+	if (inviteData.maxUses > 0 && inviteData.uses >= inviteData.maxUses) {
+		return false;
+	}
+
+	return true;
+};
+
+export const validateInviteToken = (token) => {
+	if (!token) {
+		throw new Meteor.Error('error-invalid-token', 'The invite token is invalid.', { method: 'validateInviteToken', field: 'token' });
```

Comment:
Since this is a validation method I would say to return false here rather than throw an error

Reply:
The method validates if the token is valid or not. If no token was received, I think it's an error.

Reply:
Not sure, why no token is an exception and an invalid token is a false return? Aren't they all invalid?
I'd agree to never return false, always an exception or the {invite, room}.